qiskit/tools/qi/pauli.py

```diff
@@ -41,262 +42,487 @@ class Pauli:
     Phys. Rev. A 68, 042318 – Published 20 October 2003
     """
 
-    def __init__(self, v, w):
-        """Make the Pauli class."""
-        self.numberofqubits = len(v)
-        self.v = v
-        self.w = w
+    def __init__(self, v=None, w=None, label=None):
+        r"""Make the Pauli object.
+
+        Note that, for the qubit index:
+            - Order of v, w vectors is q_0 ... q_{n-1},
+            - Order of pauli label is q_{n-1} ... q_0
+
+        E.g.,
+            - v and w vectors: v = [v_0 ... v_{n-1}], w = [w_0 ... w_{n-1}]
+            - a pauli is $P_{n-1} \otimes ... \otimes P_0$
+
+        Args:
+            v (numpy.ndarray): boolean, v vector
+            w (numpy.ndarray): boolean, w vector
+            label (str): pauli label
+        """
+        if label is not None:
+            Pauli.from_label(label)
+        else:
+            self._init_from_bool(v, w)
+
+    @classmethod
+    def from_label(cls, label):
+        r"""Take pauli string to construct pauli.
```

Comment:
what is the r here

Reply:
since there is `\` escape symbol in the docstring, I make it as raw string (r means that block is raw string)
I can remove  that.

Reply:
ok